influxdb3_write/src/catalog.rs

```diff
@@ -141,7 +141,7 @@ pub struct TableDefinition {
     pub name: String,
     #[serde(skip_serializing, skip_deserializing)]
     pub schema: Option<Schema>,
-    columns: BTreeMap<String, ColumnType>,
+    columns: BTreeMap<String, i16>,
```

Comment:
Why this change? Is ColumnType no longer public?

Reply:
I'm going to merge in for now so I can build on this, but let me know the motivation on this change.

Reply:
ColumnType does not implement `Serialize` anymore and can only be constructed via `try_from(i16)` and it can be turned into an i16. So we can serialize an i16 but not a ColumnType and thus this change.